README.md

```diff
@@ -43,13 +43,14 @@ var results = linter.verify({ source: template, moduleId: 'template.hbs' });
 ```
 
 `results` will be an array of objects which have the following properties:
-* `rule` - The name of the rule that triggered this warning/error.
-* `message` - The message that should be output.
-* `line` - The line on which the error occurred.
-* `column` - The column on which the error occurred.
-* `moduleId` - The module path for the file containing the error.
-* `source` - The source that caused the error.
-* `fix` - An object describing how to fix the error.
+
+- `rule` - The name of the rule that triggered this warning/error.
+- `message` - The message that should be output.
+- `line` - The line on which the error occurred.
+- `column` - The column on which the error occurred.
+- `moduleId` - The module path for the file containing the error.
+- `source` - The source that caused the error.
+- `fix` - An object describing how to fix the error.
```

Comment:
why did you change the syntax here?

Reply:
Sorry prettier reformatted the whole file